community/pdns/APKBUILD

```diff
@@ -39,48 +47,80 @@ build() {
 	cd "$builddir"
 
 	./configure --prefix=/usr \
-		--sysconfdir=/etc/$pkgname \
+		--sysconfdir="/etc/${pkgname}" \
 		--mandir=/usr/share/man \
 		--infodir=/usr/share/info \
 		--localstatedir=/var \
-		--libdir=/usr/lib/pdns \
+		--libdir="/usr/lib/${pkgname}" \
 		--with-modules="" \
-		--with-dynmodules="bind ldap gmysql gpgsql pipe gsqlite3 geoip" \
+		--with-dynmodules="bind geoip ldap lua mydns gmysql godbc pipe gpgsql random remote gsqlite3" \
+		--enable-tools \
+		--enable-unit-tests \
 		--disable-static
 	make
 }
 
+check() {
+	cd "$builddir"
+	PDNS_TEST_NO_IPV6=1 make check
+}
+
 package() {
 	cd "$builddir"
 
 	make DESTDIR="$pkgdir" install
 
-	install -m755 -D "$srcdir"/$pkgname.initd \
-		"$pkgdir"/etc/init.d/$pkgname
-	rm "$pkgdir"/etc/$pkgname/$pkgname.conf-dist
-	install -m600 -D "$srcdir"/$pkgname.conf \
-		"$pkgdir"/etc/$pkgname/$pkgname.conf
-	chown pdns:pdns "$pkgdir"/etc/$pkgname/$pkgname.conf
+	install -m755 -D "${srcdir}/${pkgname}.initd" \
```

Comment:
Why did you add unnecessary curly braces around variables? Please don’t do that, it just increases verbosity.

Reply:
I prefer adding the braces because it's more explicit and easier for us humans to interpret. Once you're in the habit of using quotes and braces all the time, your shell code is usually safer and less ambiguous.

```sh
printf -- 'With:\n\t%s\nWithout:\n\t%s\n\n' \
    "We're having ${fruit}s and ${nut}s after lunch." \
    "We're having $fruits and $nuts after lunch."
With:
        We're having s and s after lunch.
Without:
        We're having  and  after lunch.


fruit='bananna'
nut='pecan'

printf -- 'With:\n\t%s\nWithout:\n\t%s\n\n' \
    "We're having ${fruit}s and ${nut}s after lunch." \
    "We're having $fruits and $nuts after lunch."
With:
        We're having banannas and pecans after lunch.
Without:
        We're having  and  after lunch.


fruits='apples, oranges,'

printf -- 'With:\n\t%s\nWithout:\n\t%s\n\n' \
    "We're having ${fruit}s and ${nut}s after lunch." \
    "We're having $fruits and $nuts after lunch."
With:
        We're having banannas and pecans after lunch.
Without:
        We're having apples, oranges, and  after lunch.


nuts='peanuts'

printf -- 'With:\n\t%s\nWithout:\n\t%s\n\n' \
    "We're having ${fruit}s and ${nut}s after lunch." \
    "We're having $fruits and $nuts after lunch."
With:
        We're having banannas and pecans after lunch.
Without:
        We're having apples, oranges, and peanuts after lunch.
```

Reply:
I disagree. It may improve readability and clarity in _some cases_, but that’s not a reason for adding them _everywhere_. We have syntax highlighters in editors to highlight variables for humans. Adding curly braces around _all_ variables does not make shell scripts safer or less ambiguous, unless you don’t know what is a valid variable name in shell (for example, that variable name can never contain a space). Also changing this in existing abuilds just creates unnecessary noise.

Please follow our current code-style and do not force your habits into abuilds. Otherwise I’d have to reject your pull requests. Thank you for understanding.